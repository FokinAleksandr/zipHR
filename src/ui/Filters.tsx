import styled from '@emotion/native';

export const InputFilters = styled.View`
  background-color: rgb(133, 191, 223);
  flex-direction: row;
  padding: 12px 0 12px 12px;
`;
