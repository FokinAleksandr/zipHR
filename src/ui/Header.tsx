import styled from '@emotion/native';

export const Header = styled.View`
  justify-content: center;
  padding-left: 12px;
  height: 56px;
  width: 100%;
`;

export const HeaderTitle = styled.Text`
  font-size: 28px;
  color: white;
`;
