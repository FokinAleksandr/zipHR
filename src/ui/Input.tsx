import styled from '@emotion/native';

export const Input = styled.TextInput`
  flex: 1;
  padding: 0 8px;
  height: 40px;
  border-radius: 6px;
  background-color: white;
  margin-right: 12px;
`;
