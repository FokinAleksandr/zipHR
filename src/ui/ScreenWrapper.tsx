import styled from '@emotion/native';
import { SafeAreaView } from 'react-native-safe-area-context';

export const ScreenWrapper = styled(SafeAreaView)`
  flex: 1;
  background-color: rgb(96, 133, 247);
`;
