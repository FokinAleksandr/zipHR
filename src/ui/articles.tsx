import styled from '@emotion/native';

export const ArticleImage = styled.Image`
  width: 100px;
  height: 100px;
  margin-right: 8px;
`;

export const Articles = styled.ScrollView`
  padding: 12px;
  background-color: rgb(138, 206, 224);
`;

export const Article = styled.View`
  flex-direction: row;
  background-color: white;
  border-radius: 6px;
  padding: 12px;
  margin-bottom: 12px;
`;

export const ArticleTitle = styled.Text`
  font-size: 16px;
  font-weight: 500;
`;

export const PublishedByText = styled.Text`
  flex-shrink: 1;
  flex-wrap: wrap;
  margin: 8px 0 4px;
`;
