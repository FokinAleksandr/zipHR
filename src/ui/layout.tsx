import styled from '@emotion/native';

export const FlexGrow = styled.View`
  flex: 1;
`;
