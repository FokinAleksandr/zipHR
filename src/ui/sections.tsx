import styled from '@emotion/native';

export const SectionsWrapper = styled.View`
  background-color: rgb(236, 238, 244);
  padding-bottom: 12px;
`;

export const SectionTitle = styled.Text`
  font-size: 16px;
  font-weight: 500;
  padding: 12px;
`;

export const Sections = styled.ScrollView`
  padding: 0 12px;
`;

export const Section = styled.Pressable<{ isSelected?: boolean }>`
  border-width: 1px;
  border-color: ${props => (props.isSelected ? 'blue' : 'black')};
  border-radius: 6px;
  height: 30px;
  width: 120px;
  margin: 0 16px 12px 0;
  justify-content: center;
  align-items: center;
`;

export const SectionText = styled.Text<{ isSelected?: boolean }>`
  color: ${props => (props.isSelected ? 'blue' : 'black')};
  text-transform: capitalize;
`;
