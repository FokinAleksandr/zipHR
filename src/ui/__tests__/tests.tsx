import * as React from 'react';
import renderer from 'react-test-renderer';

import { Header } from '~/src/ui/Header';

describe('snapshot tests', () => {
  it('Header', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

export {};
