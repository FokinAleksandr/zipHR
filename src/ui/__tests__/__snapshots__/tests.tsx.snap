// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot tests Header 1`] = `
<View
  style={
    Array [
      Object {
        "height": 56,
        "justifyContent": "center",
        "paddingLeft": 12,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
